docs/src/config/pncconf.adoc

```diff
@@ -36,17 +36,17 @@ details.
 
 == Step by Step Instructions
 
-.PnCConf Splash
-image::images/pncconf-splash.png["PnCConf Configuration Wizard"]
+.PnCconf Splash
```

Comment:
Does anyone has an idea for what "PnC" stands for? I guess  @jepler can answer this...

Reply:
> Does anyone has an idea for what "PnC" stands for? I guess  @jepler can answer this...

I didn't originate this part of linuxcnc

Reply:
As it is based on Stepconf, I assumed you originated this as well. My bad. A look into the sources gives me @c-morley as possible candidate for this question :slightly_smiling_face:

Reply:
I'm guilty (of course hugely based on Jeff's way-better-coded Stepconf at the time)
Point aNd Click

Reply:
Thanks, great to have this clarified :slightly_smiling_face: